src/main/java/com/cloudbees/jenkins/plugins/bitbucket/hooks/WebhookAutoRegisterListener.java

```diff
@@ -215,11 +215,12 @@ private void removeHooks(SCMSourceOwner owner) throws IOException, InterruptedEx
                     }
                 }
                 if (hook != null && !isUsedSomewhereElse(owner, source.getRepoOwner(), source.getRepository())) {
-                    LOGGER.info(String.format("Removing hook for %s/%s", source.getRepoOwner(), source.getRepository()));
+                    LOGGER.log(Level.INFO, "Removing hook for {0}/{1}",
+                            new Object[]{source.getRepoOwner(), source.getRepository()});
```

Comment:
irrelevant for `INFO` and above, which are almost certainly logged anyway, but harmless

Reply:
JMH shows this form as faster than String.format. 

Now simple string concatenation is faster still for those cases where the message is logged, so we could argue that string concatenation should be used from WARNING up (as some - not many - users turn off INFO)